src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.html

```diff
@@ -9,6 +9,13 @@
               class="bold">Hosts</td>
           <td>{{ hostsCount }}</td>
         </tr>
+        <tr>
+          <td i18n
+              class="bold">Storage Capacity</td>
+          <td><span i18n
+                    *ngIf="filteredDevices && capacity">Number of devices: {{ filteredDevices.length }}. Raw capacity:
+            {{ capacity | dimlessBinary }}.</span></td>
```

Comment:
This doesn't seem right to me.  Maybe just `Storage capaticy: x GiB` or acc. to @pcuzner suggestion
 "just use a table showing columns for hdd, flash showing sum for the total capacity and drive count per type - so you can do a quick visual check "

Reply:
I think Something like this may work too: `2xHDD xGiB, 3xFlash yGib`
what u think? @nizamial09 @epuertat

Reply:
I think that could work. Also I believe you are taking care of this in a separate PR right @avanthakkar ? So its best to do it separately on that PR where you are following up on the review page (https://github.com/ceph/ceph/pull/42488) because I believe this PR has went through a lot to get here.

Reply:
Also I like separating them into different columns better. Its more visually catchy.

Reply:
> I think that could work. Also I believe you are taking care of this in a separate PR right @avanthakkar ? So its best to do it separately on that PR where you are following up on the review page (#42488) because I believe this PR has went through a lot to get here. Does that works for you @avanthakkar ?

I've already introduced the review section changes in my current [PR](https://github.com/ceph/ceph/pull/42600) for gather facts itself. But yeah I agree some minor things can be introduced in later PR, maybe having a general expand cluster workflow remaining cleanups.